sources/System/Renderer/RenderSystem.cpp

```diff
@@ -44,7 +47,8 @@ namespace BBM
 		this->_window.endDrawing();
 	}
 
-	void RenderSystem::onUpdate(WAL::ViewEntity<CameraComponent, PositionComponent> &entity, std::chrono::nanoseconds dtime)
+	void
```

Comment:
Is this auto-format or something you wanted? Don't you want to wrap the dtime instead?

Reply:
auto format the line was too long so the return type was split

Reply:
Wrapped the dtime instead